pkg/render/node.go

```diff
@@ -1349,9 +1349,13 @@ func (c *nodeComponent) nodeEnvVars() []corev1.EnvVar {
 	mtu := getMTU(c.cfg.Installation)
 	if mtu != nil {
 		vxlanMtu := strconv.Itoa(int(*mtu))
+		vxlanMtuV6 := strconv.Itoa(int(*mtu))
 		wireguardMtu := strconv.Itoa(int(*mtu))
+		wireguardMtuV6 := strconv.Itoa(int(*mtu))
 		nodeEnv = append(nodeEnv, corev1.EnvVar{Name: "FELIX_VXLANMTU", Value: vxlanMtu})
+		nodeEnv = append(nodeEnv, corev1.EnvVar{Name: "FELIX_VXLANMTUV6", Value: vxlanMtuV6})
 		nodeEnv = append(nodeEnv, corev1.EnvVar{Name: "FELIX_WIREGUARDMTU", Value: wireguardMtu})
+		nodeEnv = append(nodeEnv, corev1.EnvVar{Name: "FELIX_WIREGUARDMTUV6", Value: wireguardMtuV6})
```

Comment:
Do we want to set the *V6 env vars for IPv4 mode as well? In other places of this node rendering file, we check if IPv6 is enabled before setting the V6 env vars.

I don't know enough about wg and V6 but I'm wondering if there could be possible side effects in Felix where it detects `FELIX_WIREGUARDMTUV6` is set and does something related to V6 wireguard.

Reply:
Good point, I'll only add those when `FELIX_IPV6SUPPORT` is `true`. Thanks!